app/managers.py

```diff
@@ -0,0 +1,41 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.tables_actors = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.tables_actors} (first_name, last_name) VALUES (?, ?) ",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        cursor = self._connection.execute(
+            f"SELECT * FROM {self.tables_actors}"
+        )
+        return [
+            Actor(*actor) for actor in cursor
+        ]
```

Comment:
Ensure that the `Actor` class in `models.py` includes an `id` attribute, as the `all` method expects to return a list of `Actor` instances with this attribute. This is necessary for the `update` and `delete` methods to function correctly.